x/bridging_fee/ibc_middleware.go

```diff
@@ -108,6 +109,9 @@ func (im *BridgingFeeMiddleware) OnRecvPacket(ctx sdk.Context, packet channeltyp
 				EventTypeBridgingFee,
 				sdk.NewAttribute(AttributeKeyFee, fee.String()),
 				sdk.NewAttribute(sdk.AttributeKeySender, transferPacketData.Sender),
+				sdk.NewAttribute(transfertypes.AttributeKeyReceiver, transferPacketData.Receiver),
```

Comment:
AFAIU ,the receiver of this packet here is the fulfiller, which I think is misleading to put as he actually didn't pay the bridging fee.

I'd expect the receiver here to be the actual receiver who paid the fee (you can argue that the fee is paid by the sender but that's for the sender/receiver to decide) - which is the original recipient.